internal/usage/usage.go

```diff
@@ -81,6 +81,11 @@ const (
 	RSName                          = "The replica set that the index is built on."
 	Key                             = "Index keys. Should be formatted as field:type."
 	Unique                          = "Create a unique key index."
+	ResourceName                    = "Name of the resource from which to collect logs."
+	ResourceType                    = "Type of resource from which to collect logs."
+	LogTypes                        = "Array of strings specifying the types of logs to collect."
+	SizeRequestedPerFileBytes       = "Size for each log file in bytes."
+	Redacted                        = "If set to true, emails, hostnames, IP addresses, and namespaces in API responses involving this job are replaced with random string values."
```

Comment:
same as above
```suggestion
	LogRedacted                        = "If set to true, emails, hostnames, IP addresses, and namespaces in API responses involving this job are replaced with random string values."
```

Or `RedactedLog`

Reply:
Do you want me to rename also `SizeRequestedPerFileBytes`?

Reply:
Up to you, I think that one is quite unique for this command but redacted may show up in other commands and we may use it without realising it has log copy in it and not something generic